app/src/main/assets/dnsfilter.conf

```diff
@@ -28,12 +26,12 @@ maxResolverCount = 100
 # This might be usefull in case applications ignore the VPN's DNS and just use an own DNS Server,
 # such as the Googles DNS Servers
 # default = "". Uncomment setting below for using this option.
-# As an example below the list of Google DNS servers for IPV4 and IPV6.
-# routeIPs = 8.8.8.8; 8.8.4.4; 2001:4860:4860::8888; 2001:4860:4860::8844
+# As an example below the list of secure and privacy respecting DNS servers for IPV4 and IPV6.

```

Comment:
Yes, but what DNS servers?

Reply:
You posted that three times, by the way. I do think we get your point. 😛 

Are users aware that this module, if I'm understanding correctly, falls back to the router's DNS?